### ISSUE TITLE
NTR: AICA-riboside high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8812]
New term request AICA-riboside high in urine (segal_050816201451): 5-amino-4-imidazolecarboxyamide

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000117
- label: Elevated urinary AICA-riboside level
- parents: [Abnormal urine metabolite level (HP:0033354)](https://hpo.jax.org/app/browse/term/HP:0033354)
- pmid: PMID:15114530
- def: The amount of 5-amino-4-imidazolecarboxamide (AICA)-riboside in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8812
- synonyms:
